lua/bufferline/state.lua

```diff
@@ -33,7 +26,7 @@ local ANIMATION_MOVE_DURATION   = 150
 -- Section: Application state --
 --------------------------------
 
-local m = {
+local M = {
```

Comment:
`m` is commonly used as a variable, so I distinguished the module by capitalizing it.